src/content-distribution/distribute/style.scss

```diff
@@ -0,0 +1,55 @@
+@use "~@wordpress/base-styles/colors";
+
+.newspack-network-distribute {
+	height: calc(100% - 47px);
+	.components-panel {
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+	}
+	.components-panel__body {
+		border: 0;
+		&.distribute-header {
+			p {
+				color: colors.$gray-700;
+			}
+			.components-base-control__field {
+				margin: 0;
+			}
+			input[type="text"] {
```

Comment:
Instead of custom css here, I'd suggest you add `__next40pxDefaultSize` to the `TextControl` (see: https://wordpress.github.io/gutenberg/?path=/docs/components-textcontrol--docs)